include/programs.h

```diff
@@ -92,6 +94,7 @@ class Program {
 };
 
 typedef void (PROGRAMS_Main)(Program * * make);
+void PROGRAMS_Destroy(Section *);
 void PROGRAMS_MakeFile(char const * const name,PROGRAMS_Main * main);
```

Comment:
The PROGRAMS_Destroy function already exists in program.cpp which I did not change in this PR. I think it expects a Section * parameter there since it is added by AddDestroyFunction.